pkg/sqlite/movies.go

```diff
@@ -26,7 +26,7 @@ const (
 
 type movieRow struct {
 	ID       int         `db:"id" goqu:"skipinsert"`
-	Name     zero.String `db:"name"`
+	Name     string      `db:"name"`
```

Comment:
This introduces a regression where movies with blank names can be created via graphql. What is the purpose of this change? I don't see it documented in the description.